src/shared/lib/OncoKbUtils.ts

```diff
@@ -184,8 +185,52 @@ export function normalizeLevel(level:string):string|null
     }
 }
 
-export function oncogenicImageClassNames(oncogenic:string,
-                                         isVUS:boolean,
+export function oncogenicXPosition(highestSensitiveLevel: string | null) {
+    const map: { [id: string]: number } = {
+        '1': 1,
+        '2A': 2,
+        '2B': 3,
+        '3A': 4,
+        '3B': 5,
+        '4': 6
+    };
+
+    let levelIndex = highestSensitiveLevel === null ? 0 : (map[highestSensitiveLevel] || 0);
+    return -(8 + levelIndex * 30);
+}
+
+export function oncogenicYPosition(oncogenicity: string, isVUS: boolean, resistanceLevel: string | null) {
+    const oncogenicityMap: { [id: string]: number } = {
+        'Oncogenic': 0,
+        'Predicted Oncogenic': 0,
+        'Likely Oncogenic': 0,
+        'Neutral': 1,
+        'Likely Neutral': 1,
+        'Unknown': 2,
+        'Inconclusive': 2,
+        '': 4
```

Comment:
wow, empty string can be used as key?

Reply:
Yeah, it was designed that way, but latter on forgot to do some clean up. Will do as suggested below. Thanks.